code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -378,7 +378,7 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 		lit = 0 //Needed for proper update
 		update_brightness()
 		qdel(src)
-	return ..()
+	return
```

Comment:
You can actually just remove this line altogether.
Returns at the end of a proc have no purpose unless they're returning a specific value, since `return` is the default behavior when a proc reaches its end.